.gitignore

```diff
@@ -1,13 +1,9 @@
 .DS_Store
 Thumbs.db
-/input-cache/org.hl7.fhir.publisher.jar
-/input-cache/schemas
-/input-cache/schemas-r5
-/output
-/template
-/temp
-/input
-ig.ini
-package.json
-package-list.json
-SUSHI-GENERATED-FILES.md
+fsh-generated**
+input-cache/schemas
+input-cache/schemas-r5
+input-cache/publisher.jar
+output**
+template**
+temp**
```

Comment:
I think these may need a `/`, e.g., `output/**`, assuming [this is the behavior you want](https://git-scm.com/docs/gitignore):

> A trailing "/**" matches everything inside. For example, "abc/**" matches all files inside directory "abc", relative to the location of the .gitignore file, with infinite depth.